### PR TITLE
MOSIP-27354 : Create 1.2.0.1-B1_to_1.1.5.5_rollback.sql

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-authdevice-support.sql
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-authdevice-support.sql:1]: file needs to be renamed to 1.1.5.5_1.2.0.1_B1_pms-authdevice-support.sql

File: db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-regdevice-support.sql
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-regdevice-support.sql:1]: file needs to be renamed to 1.1.5.5_1.2.0.1_B1_pms-regdevice-support.sql

File: db_upgrade_scripts/mosip_pms/upgrade.properties [hunk 1]
```diff
@@ -1,4 +1,8 @@
 MOSIP_DB_NAME=
+SOURCE_DB1_NAME=mosip_authdevice
+SOURCE_DB2_NAME=mosip_regdevice
+SOURCE_DB1_SUPPORT_FILE=sql/1.2.0_pms-authdevice-support.sql
```
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/upgrade.properties:4]: file names needs to be updated

File: db_upgrade_scripts/mosip_pms/upgrade.properties [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+SOURCE_DB2_SUPPORT_FILE=sql/1.2.0_pms-regdevice-support.sql
```
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/upgrade.properties:5]: file name needs to be updated

File: db_upgrade_scripts/mosip_pms/upgrade.sh [hunk 3]
```diff
@@ -30,6 +30,11 @@ if [ "$ACTION" == "upgrade" ]; then
   UPGRADE_SCRIPT_FILE="sql/${CURRENT_VERSION}_to_${UPGRADE_VERSION}_upgrade.sql"
   if [ -f "$UPGRADE_SCRIPT_FILE" ]; then
     echo "Executing upgrade script $UPGRADE_SCRIPT_FILE"
+    if [$UPGRADE_VERSION==1.2.0]
```
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/upgrade.sh:33]: if [$UPGRADE_VERSION==1.2.0.1_B1 && $CURRENT_VERSION==1.1.5.5]

File: db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-authdevice-support.sql
Comment (balaji-alluru) [reply db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-authdevice-support.sql:1]: renamed the file as suggested

File: db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-regdevice-support.sql
Comment (balaji-alluru) [reply db_upgrade_scripts/mosip_pms/sql/1.2.0_pms-regdevice-support.sql:1]: renamed the file as suggested

File: db_upgrade_scripts/mosip_pms/upgrade.properties (hunk 1 above)
Comment (balaji-alluru) [reply db_upgrade_scripts/mosip_pms/upgrade.properties]: renamed the value of property as suggested

File: db_upgrade_scripts/mosip_pms/upgrade.sh (hunk 3 above)
Comment (balaji-alluru) [reply db_upgrade_scripts/mosip_pms/upgrade.sh]: updated as suggested

**Review by ckm007 — APPROVED** [review]

**Pull request merged**